### PR TITLE
Update comparison components and fix styles

### PR DESCRIPTION
**Pull request by cassiogreco**

### CODE REVIEW

File: src/comparison/ComparisonWidget.js [hunk 1]
```diff
@@ -25,26 +25,24 @@ export default class ComparisonWidget extends Component {
   render() {
     return (
       <div>
-          <div className="container">
```
Comment (oliverviljamaa) [inline src/comparison/ComparisonWidget.js]: So we assume there's a `container` on the page already? Highly likely, but why this change?

File: src/comparison/ComparisonWidget.js [hunk 2]
```diff
@@ -25,26 +25,24 @@ export default class ComparisonWidget extends Component {
...
-                    The true cost for sending
-                    <br/>
-                    <Select onChange={this.changeAmount} options={this.generateAmountOptions()} value={this.props.amount}/> to {this.props.target}
-                </h2>
-
-                <div className="col-xs-12 col-sm-offset-2 col-sm-8 m-t-3">
-                  <p className="small">TransferWise uses the REAL exchange rate and charges a low, transparent fee. That's why it's cheaper. It's only fair. It's your money.</p>
-                </div>
+        <div className="row text-xs-center">
+            <h2 className="col-xs-12 col-sm-12 col-xl-12">
```
Comment (oliverviljamaa) [inline src/comparison/ComparisonWidget.js]: These classes don't do anything, this `h2` should be moved out.

File: src/comparison/ComparisonWidget.js [hunk 3]
```diff
@@ -25,26 +25,24 @@ export default class ComparisonWidget extends Component {
...
+                The true cost for sending
+                <br/>
+                <Select onChange={this.changeAmount} options={this.generateAmountOptions()} value={this.props.amount}/> to {this.props.target}
+            </h2>
+
+            <div className="col-xs-12 col-sm-offset-2 col-sm-8 m-t-3">
```
Comment (oliverviljamaa) [inline src/comparison/ComparisonWidget.js]: Unnecessary `col-xs-12`.

File: src/comparison/ComparisonWidget.js (hunk 1 above)
Comment (cassiogreco) [inline src/comparison/ComparisonWidget.js]: we don't need to assume that and we don't need this class actually.

This container class is making the widget be full-sized even when the parent container has restricted width (i.e. `col-xs-6`)

Comment (oliverviljamaa) [inline src/comparison/ComparisonWidget.js]: Makes sense, we still need to assume the `container` exists on the page, as otherwise it would just be full width with no reference point.
